src/data/helpText.yaml

```diff
@@ -1,29 +1,28 @@
 product:
-  name: ""
+  name: "Name of offering (ie. your product or service)."
   version: "Provide version if available."
-  description: ""
+  description: "Describe the general offering. This should not be specific to a contract or RFP."
 author:
-  name: ""
-  company_name: ""
-  address: ""
-  email: ""
-  phone: ""
-  website: ""
+  name: "Contact name or team representing author of the ACR (optional)."
```

Comment:
Do we need to add `(optional)`? We haven't worked on making required fields yet so by default everything is optional now. I would say let the form direct the optional required and not rely on the help text?

Reply:
Ok..  Might make sense for the Legal doc as it is. Ultimately we don't want them filling that in.